### PR TITLE
Capabilities scripts that covers checks diff params settings

### PR DESCRIPTION
**Pull request by Itileda**

### CODE REVIEW

File: test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start, {rc_capabilities})
+runner.Step("RAI, PTU", commonRC.rai_ptu)
+runner.Step("Activate_App", commonRC.activate_app)
+
+runner.Title("Test")
+for _,module_name in pairs({"CLIMATE", "RADIO"}) do
```
Comment (aderiabin) [inline test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua]: @Itileda Please add space after comma:
`for _,module_name in pairs` -> `for _, module_name in pairs`

File: test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Set available control module settings SetInteriorVehicleData
+--
+-- Description:
+-- In case:
+-- 1) SDL receive only one parameter in GetCapabilites response for CLIMATE module
+-- SDL must:
+-- 1) Transfer to HMI rempote control RPCs only with supported parameter and
```
Comment (aderiabin) [inline test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua]: @Itileda  Please fix missprint in " Transfer to HMI **rempote** control"

File: test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Set available control module settings SetInteriorVehicleData
+--
+-- Description:
+-- In case:
+-- 1) SDL receive several supported Radio parameters in GetCapabilites response
+-- SDL must:
+-- 1) Transfer to HMI rempote control RPCs only with supported parameters and
```
Comment (aderiabin) [inline test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua]: @Itileda Please fix missprint in " Transfer to HMI **rempote** control"

File: test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start, {rc_capabilities})
+runner.Step("RAI, PTU", commonRC.rai_ptu)
+runner.Step("Activate_App", commonRC.activate_app)
+
+runner.Title("Test")
+for _,module_name in pairs({"CLIMATE", "RADIO"}) do
```
Comment (aderiabin) [inline test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua]: @Itileda Please add space after comma:
`for _,module_name in pairs` -> `for _, module_name in pairs`

File: test_scripts/RC/Capabilities/008_reject_request_with_both_supported_and_not_supported_parameters.lua [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start, {rc_capabilities})
+runner.Step("RAI, PTU", commonRC.rai_ptu)
+runner.Step("Activate_App", commonRC.activate_app)
+
+runner.Title("Test")
+for _,module_name in pairs({"CLIMATE", "RADIO"}) do
```
Comment (aderiabin) [inline test_scripts/RC/Capabilities/008_reject_request_with_both_supported_and_not_supported_parameters.lua]: @Itileda Please add space after comma:
`for _,module_name in pairs` -> `for _, module_name in pairs`

File: test_scripts/RC/Capabilities/009_reject_request_if_remoteControlCapabilities_parameter_was_absent_in_GetCapabilities.lua [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start, {rc_capabilities})
+runner.Step("RAI, PTU", commonRC.rai_ptu)
+runner.Step("Activate_App", commonRC.activate_app)
+--Check that all RC RPCs are rejected by SDL
+runner.Title("Test")
+for _,module_name in pairs({"CLIMATE", "RADIO"}) do
```
Comment (aderiabin) [inline test_scripts/RC/Capabilities/009_reject_request_if_remoteControlCapabilities_parameter_was_absent_in_GetCapabilities.lua]: @Itileda Please add space after comma:
`for _,module_name in pairs` -> `for _, module_name in pairs`

File: test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua [hunk 7]
```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
```
Comment (aderiabin) [inline test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua]: @Itileda Please add these tests to related test suite.

Comment (Itileda) [inline test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua]: @aderiabin , fixed in 9cf561570cbcbe2fe9be09bbd4511d2df830e9d8

File: test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua (hunk 2 above)
Comment (Itileda) [inline test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua]: @aderiabin, fixed in 9cf561570cbcbe2fe9be09bbd4511d2df830e9d8

File: test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua (hunk 1 above)
Comment (Itileda) [inline test_scripts/RC/Capabilities/006_resend_only_supported_parameter_and_reject_others.lua]: @aderiabin, fixed in 9cf561570cbcbe2fe9be09bbd4511d2df830e9d8

File: test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua (hunk 3 above)
Comment (Itileda) [inline test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua]: @aderiabin, fixed in 9cf561570cbcbe2fe9be09bbd4511d2df830e9d8

File: test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua (hunk 4 above)
Comment (Itileda) [inline test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua]: @aderiabin, fixed in 9cf561570cbcbe2fe9be09bbd4511d2df830e9d8

File: test_scripts/RC/Capabilities/008_reject_request_with_both_supported_and_not_supported_parameters.lua (hunk 5 above)
Comment (Itileda) [inline test_scripts/RC/Capabilities/008_reject_request_with_both_supported_and_not_supported_parameters.lua]: @aderiabin, fixed in 9cf561570cbcbe2fe9be09bbd4511d2df830e9d8

File: test_scripts/RC/Capabilities/009_reject_request_if_remoteControlCapabilities_parameter_was_absent_in_GetCapabilities.lua (hunk 6 above)
Comment (Itileda) [inline test_scripts/RC/Capabilities/009_reject_request_if_remoteControlCapabilities_parameter_was_absent_in_GetCapabilities.lua]: @aderiabin, fixed in 9cf561570cbcbe2fe9be09bbd4511d2df830e9d8

**Pull request merged**